### PR TITLE
2307-When-inviting-a-user-the-assigned-role-is-not-persisted

### LINKED ISSUE #2307: When inviting a user, the assigned role is not persisted.

### PR DESCRIPTION
**Pull request by mainawycliffe**
Fixes #2307

### CODE REVIEW

File: src/pages/UsersPage.tsx [hunk 1]
```diff
@@ -193,6 +178,45 @@ export function UsersPage() {
...
+            onClose={() => {
+              setInviteLink(undefined);
+            }}
+            open={!!inviteLink}
+            bodyClass=""
+            size={"small"}
+          >
+            <div className="flex flex-col bg-white p-4">
+              <div className="p-2 text-sm">
+                User has been invited successfully. An email has been sent to
```
Comment (moshloop) [inline src/pages/UsersPage.tsx:193]: Why do we have this?  The invite link is considered a password and shouldn't be shared with anyone.

Comment (moshloop) [reply src/pages/UsersPage.tsx]: Thinking about this more, I think we should keep it in

**Comment by moshloop** [comment]
@mainawycliffe - Can we a copy button ?

**Comment by mainawycliffe** [comment]
@moshloop  The copy button is there

image

**Pull request merged**